backend/src/services/group-project/group-project-service.ts

```diff
@@ -128,7 +138,7 @@ export const groupProjectServiceFactory = ({
     const projectGroup = await groupProjectDAL.transaction(async (tx) => {
       const groupProjectMembership = await groupProjectDAL.create(
         {
-          groupId: group.id,
+          groupId: group!.id,
```

Comment:
is `!` here still necessary? looks like we're already doing a null/undefined check above